build-github-tag

```diff
@@ -10,7 +10,8 @@ class TagOnRepo {
 def tagsToBuild = [
         new TagOnRepo(repo: 'linagora/james-project', tag: 'openpaas-1.4.0'),
         new TagOnRepo(repo: 'linagora/james-project', tag: 'openpaas-1.4.1'),
-        new TagOnRepo(repo: 'linagora/james-project', tag: 'james-project-3.3.0', latest: true),
+        new TagOnRepo(repo: 'linagora/james-project', tag: 'james-project-3.3.0'),
+        new TagOnRepo(repo: 'linagora/james-project', tag: 'james-project-3.4.0', latest: true),
```

Comment:
It seems that this tag is on the Apache's repo.

Reply:
Or we need to push it on the linagora repo as well?

Reply:
Good catch :)